flow/flow.go

```diff
@@ -151,26 +154,34 @@ type receiveParameters struct {
 	out    low.Rings
 	port   *low.Port
 	status []int32
+	stats  common.RXTXStats
 }
 
 func addReceiver(portId uint16, out low.Rings, inIndexNumber int32) {
 	par := new(receiveParameters)
 	par.port = low.GetPort(portId)
 	par.out = out
 	par.status = make([]int32, maxRecv, maxRecv)
-	schedState.addFF("receiver", nil, recvRSS, nil, par, nil, receiveRSS, inIndexNumber)
+	name := schedState.addFF("receiver", nil, recvRSS, nil, par, nil, receiveRSS, inIndexNumber)
+	if countersEnabled {
+		registerRXTXStatitics(&par.stats, name)
+	}
 }
 
 type receiveOSParameters struct {
 	out    low.Rings
 	socket int
+	stats  common.RXTXStats
 }
 
 func addOSReceiver(socket int, out low.Rings) {
 	par := new(receiveOSParameters)
 	par.socket = socket
 	par.out = out
-	schedState.addFF("OS receiver", nil, recvOS, nil, par, nil, sendReceiveKNI, 0)
+	name := schedState.addFF("OS receiver", nil, recvOS, nil, par, nil, sendReceiveKNI, 0)
+	if countersEnabled {
+		registerRXTXStatitics(&par.stats, name)
+	}
```

Comment:
I propose to register statistics inside addFF because it the same for all functions. Also it will remove returning name

Reply:
Ok, I made this change